src/main/actions/file.js

```diff
@@ -50,7 +51,7 @@ const forceClose = win => {
 const handleResponseForExport = (e, { type, content, filename, pathname }) => {
   const win = BrowserWindow.fromWebContents(e.sender)
   const extension = EXTENSION_HASN[type]
-  const dirname = pathname ? path.dirname(pathname) : '~'
+  const dirname = pathname ? path.dirname(pathname) : getHomePath()
```

Comment:
I think the directory of `documents` will be much better to store the default export or save files.

What do you think?

```javascript
-  const dirname = pathname ? path.dirname(pathname) : '~'
+  const dirname = pathname ? path.dirname(pathname) : getPath('documents')
```